### PR TITLE
Rename cloudProviders to platformProviders in piped examples docs

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs-dev/operator-manual/piped/installation/installing-on-cloudrun.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ description: >
         branch: {GIT_BRANCH}
 
     # Optional
-    # Enable this Piped to handle CLOUD_RUN application.
-    cloudProviders:
+    # Enable this Piped to handle CLOUDRUN application.
```
Comment (knanao) [inline docs/content/en/docs-dev/operator-manual/piped/installation/installing-on-cloudrun.md:42]: Is it better?
```suggestion
    # Enable this Piped to handle Cloud Run application.
```

File: docs/content/en/docs-v0.35.x/operator-manual/piped/installation/installing-on-cloudrun.md [hunk 2]
```diff
@@ -39,8 +39,8 @@ description: >
         branch: {GIT_BRANCH}
 
     # Optional
-    # Enable this Piped to handle CLOUD_RUN application.
-    cloudProviders:
+    # Enable this Piped to handle CLOUDRUN application.
```
Comment (knanao) [inline docs/content/en/docs-v0.35.x/operator-manual/piped/installation/installing-on-cloudrun.md:42]:
```suggestion
    # Enable this Piped to handle Cloud Run application.
```

Comment (nghialv) [reply docs/content/en/docs-v0.35.x/operator-manual/piped/installation/installing-on-cloudrun.md:42]: 👍

**Comment by nghialv** [comment]
@khanhtc1202 Did you already upgrade the Piped in play environment to the latest version?

**Comment by khanhtc1202** [comment]
> @khanhtc1202 Did you already upgrade all Pipeds in play environment to the latest version?

yes, I did 👀

**Comment by nghialv** [comment]
> yes, I did 👀

👍

File: docs/content/en/docs-dev/operator-manual/piped/installation/installing-on-cloudrun.md (hunk 1 above)
Comment (khanhtc1202) [reply docs/content/en/docs-dev/operator-manual/piped/installation/installing-on-cloudrun.md]: Nice catch, thank you 🙏

**Review by knanao — APPROVED** [review]

**Review by nghialv — APPROVED** [review]
👍

**Pull request merged**